src/fastnetmon_install.pl

```diff
@@ -1411,7 +1411,7 @@ sub apt_get {
 
     # We install one package per apt-get call because installing multiple packages in one time could fail of one package is broken
     for my $package (@packages_list) {
-        exec_command("DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes $package");
+        exec_command("DEBIAN_FRONTEND=noninteractive apt-get install -y $package");
```

Comment:
This thing is pretty tricky, we need this command to work on machines with broken apt repos or with incorrectly configured reposition. 

This option was deprecated and replace multiple commands which of them is more safe than --force-yes:
```
       --force-yes
           Force yes; this is a dangerous option that will cause apt to continue without prompting if it is doing something potentially harmful. It should not be used
           except in very special situations. Using force-yes can potentially destroy your system! Configuration Item: APT::Get::force-yes. This is deprecated and
           replaced by --allow-unauthenticated , --allow-downgrades , --allow-remove-essential , --allow-change-held-packages in 1.1.
```

I think we need only one of them:
```
 sudo apt-get -y --allow-unauthenticated  install
```

I think we need to keep --force--yes for Debian 8 and Ubuntu 14.04 and for any newer distro we need to switch to --allow-unauthenticated  instead. 

Can you add such check?